Dark_Heresy_2ed/DarkHeresy2ed.html

```diff
@@ -1645,403 +1849,205 @@
-                                <button name="roll_Lore" type="roll" 
-                                    value="/em rolls Lore:@{OtherLore} with [[(((@{OtherLoreCharacteristic} + [[(-20 + @{OtherLore1} + @{OtherLore2} + @{OtherLore3} + @{OtherLore4})]] +?{Target Modifier|0}))-1d100)/10 ]] additional degree(s) of success!">
-                                </button>
-                            </div>
-                            <div class="sheet-item" style="width:80%">
-                                <input name="attr_OtherLore" type="text">
-                            </div>
-                        </div>
-                        </td>
-                        <td><input name="attr_OtherLore1" type="checkbox" value="20"></td>
-                        <td><input name="attr_OtherLore2" type="checkbox" value="10"></td>
-                        <td><input name="attr_OtherLore3" type="checkbox" value="10"></td>
-                        <td><input name="attr_OtherLore4" type="checkbox" value="10"></td>
-                    </tr>
-                </table>
-        	</fieldset>
-            
             </div>   
-        </div>
+		</div>
+		<fieldset class="repeating_advancedskills">
+			<table>
+				<tr>
+				<td>
+				<div class="sheet-row">
+					<div class="sheet-item sheet-bigger">
+						<input class="sheet-baseinfo" name="attr_advancedskillname" type="text">
+					</div>
+					<div class="sheet-item sheet-med">
+						<select class="charaselect" name="attr_advancedskillcharacteristic">
+							<option value="@{WeaponSkill}">(WS)</option>
+							<option value="@{BallisticSkill}">(BS)</option>
+							<option value="@{Strength}">(S)</option>
+							<option value="@{Toughness}">(T)</option>
+							<option value="@{Agility}">(Ag)</option>
+							<option value="@{Intelligence}">(Int)</option>
+							<option value="@{Perception}">(Per)</option>
+							<option value="@{Willpower}">(Wp)</option>
+							<option value="@{Fellowship}">(Fel)</option>
+							<option value="@{Influence}">(Inf)</option>
+							<option value="@{Corruption}">(Cor)</option>
+							<option value="@{Insanity}">(Ins)</option>
+						</select>
+					</div>
+				</div>
+				</td>
+				<td><input name="attr_advancedskillbox1" type="checkbox" value="20"></td>
+				<td><input name="attr_advancedskillbox2" type="checkbox" value="10"></td>
+				<td><input name="attr_advancedskillbox3" type="checkbox" value="10"></td>
+				<td><input name="attr_advancedskillbox4" type="checkbox" value="10"></td>
+				<td class="roll_advancedskill"><button name="roll_advancedskill" type="roll" value="/em rolls @{advancedskillname}[[1d100]] Target [[@{advancedskill}+?{Modifier|0}]].">
+					<input disabled="true" name="attr_advancedskill" type="number" value=" -20 +(@{advancedskillbox1})+@{advancedskillcharacteristic}+@{advancedskillbox2}+@{advancedskillbox3} +@{advancedskillbox4}">
+				</button></td>
+				</tr>
+			</table>
+		</fieldset>
+		
+		<h3>Talents & Traits</h3>
```

Comment:
When you have an ampersand (&) in text in html you should use the html code: `&amp;`